EIP712/Exceptions/MemberTypeException.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace EIP712.Exceptions
+{
+    public class MemberTypeException : Eip712Exception
+    {
+
+        public MemberTypeException(string propertyName, string abiType, Type propertyType)
```

Comment:
Maybe make this optional by putting a default `null` value (in case you don't have a `Type` at the moment of throwing).